libs/packages/components/src/lib/external-link/external-link.directive.ts

```diff
@@ -98,6 +100,7 @@ export class ExternalLinkDirective implements OnChanges {
     return (
       isPlatformBrowser(this.platformId) &&
       !link.includes(location.hostname) &&
+      link.indexOf(this.emailLinkKeyword) !== 0  &&
```

Comment:
Using indexOf rather than startsWith as IE requires additional polyfill to support startsWith